src/test/resources/application.yml

```diff
@@ -25,6 +25,8 @@ spring:
 sandbox:
   cors:
     mapping: '*'
+  url:
+    edm-sorted-url: http://ftp.eanadev.org/schema_zips/edm_sorter.xsl
```

Comment:
Is it wise to have this dependency in our test suite? Perhaps it's better to have this file locally (or absent and mock the code that uses it).